build/log_shutdown.go

```diff
@@ -38,16 +39,3 @@ func (s *ShutdownLogger) Critical(v ...interface{}) {
 	s.Logger.Critical(v)
 	s.shutdown()
 }
-
-// shutdown checks whether we are listening for interrupts, since a shutdown
-// request to the signal package will block if it is not running, and requests
-// shutdown if possible.
-func (s *ShutdownLogger) shutdown() {
-	if !signal.Listening() {
-		s.Logger.Info("Request for shutdown ignored")
-		return
-	}
-
-	s.Logger.Info("Sending request for shutdown")
```

Comment:
Nice, this was a clever solution :)

Can move this log statement to where `shutdown` is called perhaps?

Reply:
Yes much better, @carlaKC takes full credit for this one. :)

I added back the log message to Critical and Criticalf. I guess that makes sense?

https://github.com/lightningnetwork/lnd/blob/4d45f50f1eb7a76b812c99aacf61b72f1c5ee7f2/build/log_shutdown.go#L28-L32

https://github.com/lightningnetwork/lnd/blob/4d45f50f1eb7a76b812c99aacf61b72f1c5ee7f2/build/log_shutdown.go#L39-L43